### PR TITLE
[IMP] module l18n_it _ipa

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

**Comment by eLBati** [comment]
:+1:

**Comment by archetipo** [comment]
:+1:

File: l10n_it_ipa/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+IPA Code (IndicePA)
+===================
+
+This module adds IPA (IndicePA) field to partner
+
+Installation
```
Comment (eLBati) [inline l10n_it_ipa/README.rst]: I think you can remove 'installation' 'configuration' and ' known issues' in this case

File: l10n_it_ipa/README.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+IPA Code (IndicePA)
+===================
+
+This module adds IPA (IndicePA) field to partner
```
Comment (eLBati) [inline l10n_it_ipa/README.rst]: Please add a link to http://www.indicepa.gov.it

File: l10n_it_ipa/README.rst (hunk 1 above)
Comment (tafaRU) [inline l10n_it_ipa/README.rst]: I used https://github.com/OCA/sale-workflow/tree/8.0/sale_order_back2draft as reference.

**Comment by nicothebrush** [comment]
:+1:

**Comment by hurrinico** [comment]
:+1:

**Pull request merged**